2009-03-25  Corentin Le Gall <clegall@proformatique.com>

	pf_manager_parking.patch
	$Revision$
	$Date$
        When parking from AMI, the callback channel is not the good one.

Index: asterisk-1.4.24/res/res_features.c
===================================================================
--- asterisk-1.4.24.orig/res/res_features.c	2009-03-25 14:08:44.000000000 +0100
+++ asterisk-1.4.24/res/res_features.c	2009-03-25 14:10:16.000000000 +0100
@@ -599,7 +599,7 @@
 	}
 
 	if (!play_announcement || !orig_chan_name) {
-		orig_chan_name = ast_strdupa(chan->name);
+		orig_chan_name = ast_strdupa(peer->name);
 	}
 
 	park_status = park_call_full(chan, peer, timeout, extout, orig_chan_name, pu);
